stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -49,6 +49,7 @@ public class MessageListViewModel(
     public val chatDomain: ChatDomain,
     private val channelId: String,
     private val messageLimit: Int = 0,
+    private val enforceUniqueReactions: Boolean = true,
```

Comment:
curious , how is this set ? since its `private`

Reply:
It's a constructor parameter